core/dbt/task/run_operation.py

```diff
@@ -66,8 +67,17 @@ def run(self) -> RunResultsArtifact:
         end = datetime.utcnow()
 
         package_name, macro_name = self._get_macro_parts()
```

Comment:
non-blocking suggestion: we don't use the `agate.Table` return value from `_run_unsafe` calls anywhere in this file, so perhaps we could return `package_name, macro_name` from `_run_unsafe` instead to avoid having to call it again here.